edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -148,7 +149,7 @@ def authenticate_credentials(self, payload):
             if attributes_updated:
                 user.save()
         except Exception as authentication_error:
-            msg = 'User retrieval failed.'
+            msg = f'[edx-drf-extensions] User retrieval failed for username {username}.'
```

Comment:
Adding `username` to the exception message to be a bit more explicit for debugging purposes.